docs/design/architecture.md

```diff
@@ -3,89 +3,103 @@ title: Architecture
 sidebar_position: 1
 ---
 
-WIP on Notion. 
+Quickwit has been designed from the ground up for cost-efficiency, scalability, and cloud-friendly. 
 
-Quickwit is built on three pillars:
-- the index: a set of data structures tailored to be queried, even on high latency storage;
-- the metastore: stores index metadata and makes them available to all search nodes;
-- the search cluster: provides high availability search, workload distribution, and efficient caching.
+Quickwit core relies on two processes:
```

Comment:
I wonder if we should say two `types of processes`. Unsure.

Reply:
Yes that’s what I wanted to say actually